sgm/inference/helpers.py

```diff
@@ -308,14 +376,16 @@ def do_img2img(
                     )
 
                 for k in c:
-                    c[k], uc[k] = map(lambda y: y[k][:num_samples].to(device), (c, uc))
+                    c[k], uc[k] = map(
+                        lambda y: y[k][:num_samples].to(device_manager.device), (c, uc)
+                    )
```

Comment:
```suggestion
                    c[k] = c[k][:num_samples].to(device_manager.device)
                    uc[k] = uc[k][:num_samples].to(device_manager.device)
```
(funny how `num_samples` is an `int` here :grin:)